core/domain/exp_domain.py

```diff
@@ -321,8 +324,11 @@ class ExplorationChange(change_domain.BaseChange):
     EXPLORATION_PROPERTIES: List[str] = [
         'title', 'category', 'objective', 'language_code', 'tags',
         'blurb', 'author_notes', 'param_specs', 'param_changes',
-        'init_state_name', 'auto_tts_enabled', 'correctness_feedback_enabled',
-        'next_content_id_index', 'edits_allowed']
+        'init_state_name', 'auto_tts_enabled',
+        'next_content_id_index', 'edits_allowed',
+        # Deprecated exploration properties.
+        DEPRECATED_EXPLORATION_PROPERTY_CORRECTNESS_FEEDBACK_ENABLED
```

Comment:
Why is this here? We did not do this for other deprecated properties.

Reply:
This is just to inform developers to avoid using this property. Other fields are marked similarly, like this:

![image](https://github.com/oppia/oppia/assets/79896602/c50cb9ae-0487-4369-a142-ccbd668b3efa)

Reply:
Ah, no, you can drop this altogether. The state properties are handled differently from exp properties since they are buried in a JSON object and sometimes a migration is needed from older schema versions of the State. If an exp property is deprecated we do not need to surface it in the domain layer at all.

Reply:
Done.